[Coffer] Fix loading songs with more than 1 chunk per track

GiR-Zippo opened the pull request:
Some chunks will be interpreted as single tracks. This will fix the issue and the chunks will be combined to its origin track.

Commit:
[Coffer] Fix loading songs with more than 1 chunk per track

MoogleTroupe commented:
if i recall, this function is used for deserializing track chunks stored in coffer, that are individual items, there would never be the case that there was more then one chunk stored in the bson passed in (unless something was being used incorrectly during serialization.

Commit:
[Transmogrify/BMPSong] remove ChannelPrefixEvents with channel > 0xF.

This have to be done, otherwise Coffer will crash, because Melanchall always expects a 4bit number for a channel.
